content/en/tracing/setup/go.md

```diff
@@ -110,112 +110,77 @@ import "gopkg.in/DataDog/dd-trace-go.v1/contrib/<PACKAGE_DIR>/<PACKAGE_NAME>"
 
 ## Configuration
 
-The tracer is configured with options parameters when the `Start` function is called. An example for generating a trace using the HTTP library:
+The Go tracer supports additional environment variables and functions for configuration.
+See all available options in the [configuration documentation][71].
+
+We highly recommend using `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services.
+Check out the [Unified Service Tagging][72] documentation for recommendations on how to configure these environment variables.
+
+You may also elect to provide `env`, `service`, and `version` through the tracer's API:
 
 ```go
 package main
 
 import (
-    "log"
-    "net/http"
-    "strings"
-
-    httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
     "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
-func sayHello(w http.ResponseWriter, r * http.Request) {
-    msg := "Hello " + strings.TrimPrefix(r.URL.Path, "/")
-    w.Write([] byte(msg))
-}
-
 func main() {
-    // start the tracer with zero or more options
-    tracer.Start(tracer.WithServiceName("test-go"))
+    tracer.Start(
+        tracer.WithEnv("prod"),
+        tracer.WithService("test-go"),
+        tracer.WithVersion("abc123"),
+    )
     defer tracer.Stop()
-
-    mux := httptrace.NewServeMux() // init the http tracer
-    mux.HandleFunc("/", sayHello) // use the tracer to handle the urls
-
-    err := http.ListenAndServe(":9090", mux) // set listen port
-    if err != nil {
-        log.Fatal("ListenAndServe: ", err)
-    }
```

Comment:
Is there a reason we're getting rid of the example? Do we think a complete example isn't valuable here?

Reply:
I had a couple reasons:
1. This example is only useful for folks instrumenting an HTTP application.
2. It's lengthy in the sense that there's a lot of setup here which isn't directly related to setting up the tracer.

I thought we could condense the code example to be more directly about creating/starting the tracer. Additionally, we don't have such code examples in our other tracer docs. They tend to be more concise.

Reply:
👍 Ok, that makes sense to me.